src/views/general-partners.njk

```diff
@@ -0,0 +1,25 @@
+{% extends "layout.njk" %}
+
+{% block beforeContent %}
+  {% include "includes/back-link.njk" %}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+ 
+        {% if (data['registrationOrTransition'] == "registration") %}
```

Comment:
we don't have this data

I think Vernon said to remove this caption

Reply:
Removed.